main.c

```diff
@@ -160,6 +159,8 @@ int WINAPI _tWinMain (HINSTANCE hThisInstance,
   PrintDebug(_T("Shell32.dll version: 0x%lx"), shell32_version);
 #endif
 
+  /* Parse command-line options */
+  ProcessCommandLine(&o, GetCommandLine());
```

Comment:
I'm not sure if this absolutely correct to move this function. Can somebody please confirm it won't break anything?